pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: layout.pug
+navigationTitle: Workspace Catalog Applications
+title: Workspace Catalog Applications
+menuWeight: 20
+excerpt: Workspace Catalog Applications
+---
+<!-- TODO: settle down the link target to custom applications -->
+Catalog applications are any third-party or open source applications that appear in the Catalog. These applications can be provided by D2iQ for use in your environment.
+
+## Install the workspace catalog via the CLI
+
+Follow these steps to install the catalog from the CLI.
+
+1.  Set the `WORKSPACE_NAMESPACE` environment variable to the name of your workspace’s namespace:
+
+    ```bash
+    export WORKSPACE_NAMESPACE=<workspace namespace>
+    ```
+
+1.  Create the `GitRepository`:
+
+    ```sh
+    kubectl apply -f - <<EOF
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
+      labels:
+        kommander.d2iq.io/gitapps-gitrepository-type: catalog
+        kommander.d2iq.io/gitrepository-type: catalog
+    spec:
+      interval: 1m0s
+      ref:
+        tag: v2.1.1
+      timeout: 20s
+      url: https://github.com/mesosphere/dkp-catalog-applications  
+    EOF
+    ```
+
+1.  Verify that you can see the workspace catalog `Apps` available in the UI and in the CLI, using `kubectl`:
```

Comment:
I think this was my initial change anyway - but I think a comma here shows that they are two separate actions that do the same thing. So just committing here.

```suggestion
1.  Verify that you can see the workspace catalog `Apps` available in the UI, and in the CLI, using `kubectl`:
```